### DISCUSSION TITLE
Android example app using whisper.cpp from ggerganov

### DISCUSSION THREAD

**Discussion by Digipom** [openai/whisper#691] [Show and tell]
Example app: https://github.com/Digipom/WhisperCppAndroidDemo

Performance is pretty good with the tiny and base models. Thank you to @ggerganov for porting Whisper to C++.

**Comment by UnstoppableCurry** [comment]
nice it's work on my phone

**Reply by bilalazhar72** [reply]
is there a way to run it

**Comment by ivanmladek** [comment]
works

**Comment by pavan901** [comment]
now its not working please provide latest version that compitable with latest NDK and also attach readme file if possible
